packages/medusa/src/api/routes/store/variants/list-variants.ts

```diff
@@ -52,6 +53,14 @@ export default async (req, res) => {
     selector = { id: ids.split(",") }
   }
 
+  if (inventory_quantity) {
+    selector.inventory_quantity = inventory_quantity
+  }
+
+  if (title) {
+    selector.title = title
+  }
```

Comment:
nit: Is there a reason to why we are not using the `includeFields` approach as seen in other controllers?